config/core/test-configs.yaml

```diff
@@ -1318,6 +1323,20 @@ device_types:
     <<: *qemu_x86_64
     params: *uefi_i386
 
+  r8a7743-iwg20d-q7:
+    mach: renesas
+    class: arm-dtb
+    boot_method: uboot
+    filters:
+      - passlist: {defconfig: ['defconfig']}
+
+  r8a7745-iwg22d-sodimm:
```

Comment:
Not in upstream LAVA https://git.lavasoftware.org/lava/lava/-/tree/master/etc/dispatcher-config/device-types

Reply:
https://git.lavasoftware.org/lava/lava/-/blob/master/etc/dispatcher-config/device-types/r8a7745-iwg22d-sodimm.jinja2

Reply:
Failing to boot: https://staging.kernelci.org/test/plan/id/618a34329f7684a65aa6c548/
Corresponding LAVA job: https://lava.ciplatform.org/scheduler/job/514410

Reply:
This may actually be a kernel problem.  Is this platform known to work with some stable kernel version?

Reply:
This one has now been offline for about 2 weeks in the lab, and before that it was still failing to boot although the kernel did start:
https://lava.ciplatform.org/scheduler/job/594763
@patersonc @aliceinwire Any update on these issues?